runtime/composable/src/lib.rs

```diff
@@ -0,0 +1,930 @@
+
+			return (list, storage_info)
+		}
+
+		fn dispatch_benchmark(
+			config: benchmarking::BenchmarkConfig
+		) -> Result<Vec<benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
+			use benchmarking::{Benchmarking, BenchmarkBatch, add_benchmark, TrackedStorageKey};
+
+			use system_benchmarking::Pallet as SystemBench;
+			impl system_benchmarking::Config for Runtime {}
+
+			use session_benchmarking::Pallet as SessionBench;
+			impl session_benchmarking::Config for Runtime {}
+
+			let whitelist: Vec<TrackedStorageKey> = vec![
+				// Block Number
+				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef702a5c1b19ab7a04f536c519aca4983ac").to_vec().into(),
+				// Total Issuance
+				hex_literal::hex!("c2261276cc9d1f8598ea4b6a74b15c2f57c875e4cff74148e4628f264b974c80").to_vec().into(),
+				// Execution Phase
+				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7ff553b5a9862a516939d82b3d3d8661a").to_vec().into(),
+				// Event Count
+				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850").to_vec().into(),
+				// System Events
+				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
+			];
+
+			let mut batches = Vec::<BenchmarkBatch>::new();
+			let params = (&config, &whitelist);
+
+			add_benchmark!(params, batches, frame_system, SystemBench::<Runtime>);
+			add_benchmark!(params, batches, balances, Balances);
+			add_benchmark!(params, batches, timestamp, Timestamp);
+			add_benchmark!(params, batches, session, SessionBench::<Runtime>);
+			add_benchmark!(params, batches, collator_selection, CollatorSelection);
+			add_benchmark!(params, batches, indices, Indices);
+			add_benchmark!(params, batches, membership, CouncilMembership);
+			add_benchmark!(params, batches, treasury, Treasury);
+			add_benchmark!(params, batches, scheduler, Scheduler);
+			add_benchmark!(params, batches, democracy, Democracy);
+			add_benchmark!(params, batches, collective, Council);
+			add_benchmark!(params, batches, crowdloan_bonus, LiquidCrowdloan);
+			add_benchmark!(params, batches, utility, Utility);
+
+			if batches.is_empty() { return Err("Benchmark not found for this pallet.".into()) }
+			Ok(batches)
+		}
+	}
+}
+
+struct CheckInherents;
+
+impl cumulus_pallet_parachain_system::CheckInherents<Block> for CheckInherents {
+	fn check_inherents(
+		block: &Block,
+		relay_state_proof: &cumulus_pallet_parachain_system::RelayChainStateProof,
+	) -> sp_inherents::CheckInherentsResult {
+		let relay_chain_slot = relay_state_proof
+			.read_slot()
+			.expect("Could not read the relay chain slot from the proof");
+
+		let inherent_data =
+			cumulus_primitives_timestamp::InherentDataProvider::from_relay_chain_slot_and_duration(
+				relay_chain_slot,
+				sp_std::time::Duration::from_secs(6),
+			)
+			.create_inherent_data()
+			.expect("Could not create the timestamp inherent data");
+
+		inherent_data.check_extrinsics(block)
+	}
+}
+
```

Comment:
could we have some document on how to maintain 2 runtimes? who will do it? may be reference on how other teams do it. 

could we share some setup code amid 2?

Reply:
the [polkadot repo](https://github.com/paritytech/polkadot/tree/master/runtime) maintains 4 runtimes. Best practices are lifted from there.